docs/source/methods/dinov2.md

```diff
@@ -32,11 +32,6 @@ if __name__ == "__main__":
         data="my_data_dir",
         model="dinov2_vit/vitb14_pretrain",
         method="dinov2",
-        method_args={
-            # Only set these arguments when starting from a pretrained model
-            "student_freeze_backbone_epochs": 1,  # Freeze the student backbone for 1 epoch
-            "student_freeze_last_layer_epochs": 0,  # Unfreeze the student last layer
-        },
```

Comment:
Should we add here a `epochs=100      # We recommend epochs = 125000 * batch size / dataset size`

As a follow-up we should set `epoch = "auto"` by default and set it automatically for DINOv2.